### DISCUSSION TITLE
A lot of warnings in console after upgrading nollup to 0.14.3

### DISCUSSION THREAD

**Discussion by frederikhors** [urql-graphql/urql#1234] [Q&A]
After upgrading from `nollup 0.14.2` to `0.14.3` I get a lot of warning now, why?

URQL only.

I'm using https://github.com/rixo/svelte-template-hot by the amazing @rixo.

![image](https://user-images.githubusercontent.com/41120635/102099536-ab63f200-3e28-11eb-83fd-70b5283aa9c3.png)

[Answer from `nollup`'s creator](https://github.com/PepsRyuu/nollup/issues/166#issuecomment-744519863):

> Not entirely sure how that's happening as a result of this change. But usually those happen because the library has relative paths to a source map included in the library. Nollup doesn't try to remove those comments. In the case of @urql/core, it has the following inside it:
> 
> ```
> //# sourceMappingURL=urql-core.mjs.map
> ```
> 
> I'm not sure how that would have worked before though. As far as I understand, other bundlers like Webpack would still show the same warning message. There's a plugin for this in Webpack, I suppose something similar would be needed here: https://webpack.js.org/loaders/source-map-loader/

Can we do something?

❤️

**Comment by kitten** [comment]
We use sourcemaps and nollup is probably not serving them correctly anymore.
I'm not sure why you're posting that here frankly because I'm not sure whether we can help you with that 😅 

But judging from the warning, Nollup is not serving our sourcemaps anymore

**Reply by kitten** [reply]
Oh actually, could be that the author of Nollup isn't aware of this: https://www.npmjs.com/package/rollup-plugin-sourcemaps
I assumed they wouldn't do much bundling for dependencies in dev but if they do then you may want this plugin to mute the warnings and use the source maps.

**Reply by frederikhors** [reply]
I don't know. I'm trying to understand,
